### PR TITLE
Bugman ninja knives

### PR DESCRIPTION
**Pull request by ErrorsWindows**
## What this does
Throwing weapons box traitor item for insectoids

## Why it's good
I'm feeling sufficiently American this morning to concede the point that [throwing knives are badass](https://youtu.be/8PVS1a7Bp9E?t=4)

I mean just think about it, You're watching as two young assistants steal a head of personnel's bag, but the paper pusher resists much more than expected and their bag tears open and the contents of the bag fall onto the ground, as the robbers shuffle around in the following chaos, leaving you awestruck at what to do, all of a sudden the shaft miner rolls up and lets the throwing knives fly and absolutely NAILS the two hooligans to the next airlock, NOBODY was expecting that

Also things just aren't the same ever since they nofun'd the ninja stars

### But why for insectoids?
It is a fact that all Turks know the ninja arts by genetic nature... Just think about it, Skanderbeg was an honorary Turk for half his life and went on to single-handedly halt the Ottoman expansion for 20 years until his death

image

## Changelog
:cl:
 * rscadd: Throwing knives box, traitor item for insectoid

### CODE REVIEW

**Comment by Pastamadness** [comment]
Ok but why bugs

**Comment by angryturnip** [comment]
Doesn't really make sense to be a bug thing. I like it as something to buy via uplink. I just don't understand the correlation to bugmen. If you tweaked it a bit so the knives were made or chitin or something I think it would fit better.

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -582,6 +582,14 @@ var/list/uplink_items = list()
 	cost = 6
 	discounted_cost = 4
 	jobs_with_discount = list("Grey")
+	
+/datum/uplink_item/turkmen/throwingbox
```
Comment (gurfan) [inline code/datums/uplink_item.dm]: `/datum/uplink_item/turkmen` doesn't exist.

**Comment by wotisjanitor** [comment]
soulful clock art

**Comment by ErrorsWindows** [comment]
1. yall are acting like bugmen had any flavor to begin with
2. ok whoops, in a moment

**Comment by ErrorsWindows** [comment]
also, writing the pull request description is the last thing that goes on in a pull request so I wasn't really putting my brain cells together for it at the moment
the thing with ninja weapons is that they're [typically cheaply made](https://www.karatemart.com/throwing-knives?P=all) (by nature) and the stereotype with bugmen is that they're cheaply made as well (also by nature, I guess)

**Comment by SECBATON-GRIFFON** [comment]
i think some other servers already have throwing knives as a generic traitor item

**Comment by Kurfursten** [comment]
Lack of support, but don't give up young coder. Maybe take feedback and try something else.

**Pull request closed without merging**